domain-models-runtime/src/test/java/org/folio/rest/persist/PostgresClientTransactionsIT.java

```diff
@@ -243,4 +243,42 @@ public void test(TestContext context) {
     deleteTransaction(context);
   }
 
+  @Test
+  public void testWithTransactionSuccess(TestContext context) {
+    PostgresClient c1 = PostgresClient.getInstance(vertx, tenant);
+
+    c1.withTransaction(f ->
+      f.query("INSERT INTO " + fullTable + " VALUES (2, '{ \"name\": \"a1\"}');\n").execute().map("inserted")
+    ).onComplete(context.asyncAssertSuccess(res -> context.assertEquals("inserted", res)));
+  }
```

Comment:
All four tests should check the key feature of the new method:
* If the connection automatically been closed in all cases?
* Is the transaction automatically been closed (commit or rollback) in all cases?